### ISSUE TITLE
Does alerts support sending Feishu?

### ISSUE THREAD

**Issue by yxiaoy6** [SigNoz/signoz#2643]
## Is your feature request related to a problem?

*Please describe.*

## Describe the solution you'd like

Does alerts support sending Feishu?

**Comment by srikanthccv** [comment]
No, it is not supported. However, you can send them to some webhook service and then forward them to your destination app.

**Comment by yxiaoy6** [comment]
> No, it is not supported. However, you can send them to some webhook service and then forward them to your destination app.

i see，thanks

**Comment by yxiaoy6** [comment]
![image](https://user-images.githubusercontent.com/43592152/236176999-01034727-f0b8-4b08-90aa-65b7d0440d09.png)
Can you help me look at this? I set the threshold to be above 3, and it is at least once, but the alarm status of this rule is always ok, and it is not triggered.

**Comment by srikanthccv** [comment]
That shouldn't be the case. It takes some time for the status to change.

**Comment by yxiaoy6** [comment]
> That shouldn't be the case. It takes some time for the status to change.

It has not been triggered so far yesterday

**Comment by srikanthccv** [comment]
>It has not been triggered so far yesterday

Are you talking about the status change in UI or the alert not getting triggered?
